src/vs/workbench/parts/output/browser/output.contribution.ts

```diff
@@ -45,3 +29,25 @@ registerMode({
 	moduleId: 'vs/workbench/parts/output/common/outputMode',
 	ctorName: 'OutputMode'
 });
+
+// Register Output Panel
+(<panel.PanelRegistry>platform.Registry.as(panel.Extensions.Panels)).registerPanel(new panel.PanelDescriptor(
+	'vs/workbench/parts/output/browser/outputPanel',
+	'OutputPanel',
+	OUTPUT_PANEL_ID,
+	nls.localize('output', "Output"),
+	'output',
+	40
```

Comment:
@isidorn I dont think we need the "order" property for panels, it is used for showing the actions in the activity bar only